src/TriangleNodesProvisioner.cpp

```diff
@@ -399,15 +399,18 @@ namespace blitzdg {
 
         const real_vector_type& r = *rGrid;
         const real_vector_type& s = *sGrid;
-        const real_matrix_type& V2D = *V;
+        real_matrix_type& V2D = *V;
 
         index_type Np = NumLocalPoints;
         index_type K = NumElements;
 
         real_matrix_type V2Dr(Np,Np), V2Ds(Np,Np);
         
-        real_matrix_type& D_r = *Dr;
-        real_matrix_type& D_s = *Ds;
+        real_matrix_type D_r = *Dr;
+        real_matrix_type D_s = *Ds;
```

Comment:
Is your intent to be making copies here?

Reply:
Nope, thanks for noticing. Fixed.